src/Inputs/ComboBox.tsx

```diff
@@ -0,0 +1,294 @@
+import React, {
+    useState,
+    useEffect,
+    useMemo,
+    useCallback,
+    Children,
+    isValidElement,
+} from 'react';
+import styled, { withTheme, DefaultTheme } from 'styled-components';
+import {
+    flex,
+    scroll,
+    position,
+    transition,
+    clickable,
+    styledCondition,
+    darken,
+} from '@Utils/Mixins';
+import { useTransition } from '@Utils/Hooks';
+import { LabelLayout, LabelLayoutProps,InputFragment } from '@Layouts';
+
+const SPEED = 'normal';
+
+let myWindow = window as any;
+
+export interface SelectProps extends LabelLayoutProps {
+    disabled?: boolean;
+    placeholder?: string;
+    value?: string | number;
+    theme: DefaultTheme;
+    onChange?: Function;
+    limit?: number;
+    NUM_VISIBLE_SELECTIONS:number;
+}
+
+const _Select: React.FC<SelectProps> = ({
+    disabled,
+    value,
+    children,
+    limit = 4,
+    placeholder = '',
+    onChange = (): void => {},
+    theme,
+    name,
+    NUM_VISIBLE_SELECTIONS,

```

Comment:
This should be a variable not a constant?

Meaning
, numVisible = NUM_VISIBLE_SELECTIONS

and at line 21, const NUM_VISIBLE_SELECTIONS = 3;

Reply:
I think I did it as mentioned now